src/components/Calculator.vue

```diff
@@ -1,168 +1,320 @@
+                    <button class="btn-grey" value="fact">x!</button>
+                    <button class="btn-grey" value="inv">Inv</button>
+                    <button class="btn-grey" value="sin">sin</button>
+                    <button class="btn-grey" value="in">ln</button>
+                    <button class="btn-grey" value="pi">pi</button>
+                    <button class="btn-grey" value="cos">cos</button>
+                    <button class="btn-grey" value="log">log</button>
+                    <button class="btn-grey" value="e">e</button>
+                    <button class="btn-grey" value="tan">tan</button>
+                    <button class="btn-grey" value="sqrt">sqrt</button>
+                    <button class="btn-grey" value="ans">Ans</button>
+                    <button class="btn-grey" value="exp">EXP</button>
+                    <button class="btn-grey" value="sqr">sqr</button>
+                </div>
+                <div class="standard-block">
+                    <button class="btn-grey" value="(">(</button>
+                    <button class="btn-grey" value=")">)</button>
+                    <button class="btn-grey" value="%">%</button>
+                    <button class="btn-grey" value="AC" @click="onClearScreen()">CE</button>
+                    <button class="btn-light" value="7" @click="onDigitClick('7')">7</button>
+                    <button class="btn-light" value="8" @click="onDigitClick('8')">8</button>
+                    <button class="btn-light" value="9" @click="onDigitClick('9')">9</button>
+                    <button class="btn-grey" value="/" @click="onOperationClick('/')">/</button>
+                    <button class="btn-light" value="4" @click="onDigitClick('4')">4</button>
+                    <button class="btn-light" value="5" @click="onDigitClick('5')">5</button>
+                    <button class="btn-light" value="6" @click="onDigitClick('6')">6</button>
+                    <button class="btn-grey" value="*" @click="onOperationClick('*')">*</button>
+                    <button class="btn-light" value="1" @click="onDigitClick('1')">1</button>
+                    <button class="btn-light" value="2" @click="onDigitClick('2')">2</button>
+                    <button class="btn-light" value="3" @click="onDigitClick('3')">3</button>
+                    <button class="btn-grey" value="-" @click="onOperationClick('-')">-</button>
+                    <button class="btn-light" value="0" @click="onDigitClick('0')">0</button>
+                    <button class="btn-grey" value="." @click="onDotClick()">.</button>
+                    <button class="btn-blue" value="=" @click="onGetResult()">=</button>
+                    <button class="btn-grey" value="+" @click="onOperationClick('+')">+</button>
+                </div>
+            </div>
+            <p v-for="(operation, index) in history" v-bind:key="index">{{ operation }}</p>
```

Comment:
you have separate route for history in your app header. shouldn't it be there?

Reply:
yes, in next pull it will be. I used it here just for testing - to see if it correct add expressions to history